README.md

```
@@ -1,51 +1,169 @@
+    report_prefix="enriched",
+    task_name="enrichment",
+)
+```
+
+Async API: `run_main_process` in `deep_research.main_process` (same parameters as `execute_main_process`).
 
-### DeepResearch Bench Leaderboard Screenshot
+## Project layout
 
-<img width="1178" height="751" alt="huggingface_leaderboard" src="https://github.com/user-attachments/assets/2d88256a-5e77-46f8-bd51-fe083bfcc780" />
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)